.github/pull_request_template.md

```diff
@@ -20,25 +21,39 @@ Please explain **IN DETAIL** what the changes are in this PR and why they are ne
 - [ ] I have written necessary rustdoc comments
 - [ ] I have added necessary unit tests and integration tests
 - [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
-- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer the issue)
+- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
 - [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
 
+<!-- 
+
+You can ignore or delete the section below if your PR doesn't contain user-facing changes
+
+-->
+
 ## Documentation
 
-If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.
+- [ ] My PR contains user-facing changes!
```

Comment:
A checkbox must be checked before merging, which is required by `task-list-completed`

Reply:
So I removed the requirement 😇 I don't think the restriction is very useful. If reviewers are happy, it should mean the task are done.

Reply:
Let me reverse the tick 🤪